flink-runtime/src/test/java/org/apache/flink/runtime/operators/testutils/DummyEnvironment.java

```diff
@@ -190,4 +191,8 @@ public InputGate getInputGate(int index) {
 		return null;
 	}
 
+	@Override
+	public TaskEventDispatcher getTaskEventDispatcher() {
+		return null;
```

Comment:
`throw new UnsuportedOperationException()` ?

Reply:
ok, let's try this out